content/en/user-guide/aws/elb/index.md

```diff
@@ -148,6 +148,15 @@ The following output will be retrieved:
 }
 ```
 
+#### Alternative URL structure
+
+If a request cannot be made to a subdomain of `localhost.localstack.cloud`, an alternative URL structure is available, however it is not returned by AWS management API methods.
+To make a request against an ELB with id `<elb-id>`, use the URL:
+
+```bash
+http(s)://localhost.localstack.cloud:4566/_aws/elb/<elb-id>/<elb-path>
+```
```

Comment:
Might be worth having an real world example of the equivalent between the host based type and the path based:

> Here's an example of how you would access the load balancer with a name of `elb-0v1p6q6` with the regular URL format:
>
> `http(s)://elb-0v1p6q6.elb.localhost.localstack.cloud:4566/test/path`
> With the alternative URL structure:
> `http(s)://localhost.localstack.cloud:4566/_aws/elb/elb-0v1p6q6/test/path`

Exact test to be updated, but just throwing the idea.

Reply:
Thanks @bentsku , that's a great suggestion. I have implemented in defed437ad9ba0b9e3e9022888ed07366eb365de